handlers/main.yml

```diff
@@ -9,6 +9,35 @@
     loop_var: restart_item
   listen: "Reload systemd units"
 
+- name: Enable systemd units
+  become: true
+  systemd:
+    name: "{{ enable_item.dest | basename }}"
+    daemon_reload: true
+    enabled: "{{ enable_item.unit_item.enabled | default(_default_unit_enabled) }}"
+    state: "{{ enable_item.unit_item.state | default(_default_unit_state) }}"
+  with_items: "{{ restart_units.results }}"
+  loop_control:
+    loop_var: enable_item
+  listen: "Enable systemd units"
+  when:
+    - uninstall_item.state is not defined or uninstall_item.state is defined and uninstall_item.state != "absent"
+
+- name: Uninstall systemd units
+  become: true
+  systemd:
+    name: "{{ enable_item.dest | basename }}"
```

Comment:
=> uninstall_item.dest

Reply:
Honestly, this is ridiculous, I think I created the PR too early. I'm on it.